client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -49,6 +53,17 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		isBusy,
 		trackingEventsProps,
 	} = props;
+
+	const importSiteQueryParam = getQueryArg( window.location.href, 'from' )?.toString() || '';
+	const isMigrationModalFeatureEnabled = config.isEnabled( 'migration_assistance_modal' );
+	const showModal =
+		isMigrationModalFeatureEnabled &&
```

Comment:
No need to declare a variable for this, let's simply use `config.isEnabled` :)

Reply:
Not sure if this will improve the code, I like having the variable declared. Looks more readable to me, this will be removed anyway later once we go fully live. 👍  
Thank you 🙇